docs/source/user/first-look-at-DP.ipynb

```diff
@@ -0,0 +1,649 @@
+    }
+   ],
+   "source": [
+    "count_meas.map(d_in=max_contributions)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The privacy map tells us that passing data through this transformation will incur an $\\epsilon = 0.5$ privacy spend to any individual in a dataset passed in.\n",
+    "$\\epsilon$ is a commonly-used proxy to quantify the worst-case risk to any individual.\n",
+    "It is customary to refer to a data release with such bounded risk as $\\epsilon$-DP.\n",
+    "\n",
+    "In this case, the privacy spend is based on two factors: the noise scale, and the max contributions. \n",
+    "In other cases, this number may also vary based on other parameters used to make any of the other transformations in the chain."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Differentially Private Release\n",
+    "\n",
+    "Notice that all the work we've done thus far has never involved looking at the sensitive dataset we're analyzing.\n",
+    "In order to obtain accurate privacy guarantees, the OpenDP Library should mediate all access to the sensitive dataset.\n",
+    "\n",
+    "We now invoke the measurement on our dataset, and consume $\\epsilon = 0.5$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "7001"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "csv_data = open(\"../data/teacher_survey/teacher_survey.csv\").read()\n",
+    "dp_count = count_meas(csv_data)\n",
+    "dp_count"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result is a random draw from the discrete laplace distribution, centered at the true count of the number of records in the underlying dataset.\n",
+    "\n",
+    "We can also give a $(1 - \\alpha)100$% confidence interval for this release:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "6.42960493986766"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from opendp.accuracy import discrete_laplacian_scale_to_accuracy\n",
+    "discrete_laplacian_scale_to_accuracy(scale=2., alpha=0.05)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When the discrete Laplace distribution's scale is 2, the DP estimate differs from the true value by no more than 6.43, at a 95% confidence level."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Accuracy vs. Utility\n",
```

Comment:
```suggestion
    "## 5. Accuracy vs. Utility\n",
```

Reply:
I more clearly delineated the steps of the process. This section was meant to be after the procedure.